command/ca/provisioner/caConfigClient.go

```diff
@@ -19,62 +19,66 @@ func newNoDB() *nodb {
 	return &nodb{}
 }
 
-func (n *nodb) CreateProvisioner(ctx context.Context, prov *linkedca.Provisioner) error {
+func (n *nodb) CreateProvisioner(context.Context, *linkedca.Provisioner) error {
 	return nil
 }
 
-func (n *nodb) GetProvisioner(ctx context.Context, id string) (*linkedca.Provisioner, error) {
+func (n *nodb) GetProvisioner(_ context.Context, id string) (*linkedca.Provisioner, error) {
+	_ = id
```

Comment:
This might make more sense.
```suggestion
func (n *nodb) GetProvisioner(context.Context, string) (*linkedca.Provisioner, error) {
```